src/clr-addons/generic-quick-list/generic-quick-list.ts

```diff
@@ -67,6 +70,9 @@ export class ClrGenericQuickList<T extends ClrGenericQuickListItem> implements O
   }
 
   setFocusOnAdd(): void {
+    if (this.destroyed)
```

Comment:
Could you instead please remember the return of setTimeout and use clearTimeout inside ngOnDestroy?
Would be more future proof if any method gets added inside setTimeout